tools/openshift-ci/yum.repos.d/aos-devel.repo

```diff
@@ -0,0 +1,83 @@
+[aos-devel-4.14]
+name=aos-devel-4.14
+baseurl=https://ocp-artifacts.hosts.prod.psi.rdu2.redhat.com/pub/RHOCP/plashets/4.14/stream/el8/latest/x86_64/os/
```

Comment:
We may get security alert as this expose internal info to outside of Red Hat.

Reply:
ok thanks @liangxia